File: back-end/ReportingTasksProject/01_BOL/DetailsWorkerInProjects.cs
using BOL;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace _01_BOL
{
  public  class DetailsWorkerInProjects
    {
        public int UserId { get; set; }
        public string TeamLeaderName { get; set; }
        public string Name { get; set; }
        public string Kind { get; set; }
        public int Hours { get; set; }
        public List<ActualHours> ActualHours { get; set; }
       
    }
}

File: back-end/ReportingTasksPhp/Models/DetailsWorkerInProjects.php
<?php


class DetailsWorkerInProjects implements JsonSerializable {
    public $UserId;
    public $TeamLeaderName;
    public $Name;
    public $Kind;
    public $Hours;    
    public $ProjectId;
    public $ActualHours;
    public function __construct($sqlRaw_) {    
         $this->UserId = $sqlRaw_['user_id'];
         $this->TeamLeaderName = $sqlRaw_['teamLeadername']; 
         $this->Name = $sqlRaw_['user_name']; 
         $this->Kind = $sqlRaw_['user_kinds_name']; 
         $this->Hours = $sqlRaw_['hours']; 
         $this->ProjectId=$sqlRaw_["project_id"];
         $id=$sqlRaw_["project_id"];
         $userId=$sqlRaw_["user_id"];
              $query =  "SELECT * FROM tasks.actual_hours where project_id='$id' and user_id='$userId'";
            $this->ActualHours=db_access::run_reader($query, "ActualHours");   
    }
    
    
    
    public function jsonSerialize() {
          return get_object_vars($this);
    }

}
